core/store/orm/config.go

```diff
@@ -415,12 +415,12 @@ func (c Config) ExplorerSecret() string {
 	return c.viper.GetString(EnvVarName("ExplorerSecret"))
 }
 
-// OracleContractAddress represents the deployed Oracle contract's address.
-func (c Config) OracleContractAddress() *common.Address {
-	if c.viper.GetString(EnvVarName("OracleContractAddress")) == "" {
-		return nil
+// OperatorContractAddress represents the deployed Oracle contract's address.
```

Comment:
Could we have a slightly more detailed explanation here?

Reply:
@thodges-gh any suggestions on a more detailed explanation?

Reply:
Operator.sol is the next version of the Oracle.sol contract that every node operator needs to deploy to accept RunLog requests.